app/components/Comments/CommentView.tsx

```diff
@@ -93,23 +92,26 @@ const CommentView = (props: Props) => {
           triggerComponent={
             <Icon
               size={20}
-              className={styles.sortIcon}
+              className="secondaryFontColor"
```

Comment:
What is this? Do we have global css classes?

Reply:
Yes we've had this for like 2 years now